api/src/main/java/jakarta/json/bind/customization/Scope.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2021 Oracle and/or its affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+
+package jakarta.json.bind.customization;
+
+/**
+ * Scope of upon which the customization should be used.
+ */
+public enum Scope {
```

Comment:
I recommend using more specific name, so this doesn't creep into autocomplete for e. g. `jakarta.inject.Scope`.
`CustomizationScope` would be more fitting.

Reply:
Yes, this seems reasonable to me. Original thought was to avoid having long enum name